src/dvb.c

```diff
@@ -1700,9 +1700,10 @@ void get_signal(adapter *ad, int *status, uint32_t *ber, uint16_t *strength,
 // returns the strength and SNR between 0 .. 65535
 
 int get_signal_new(adapter *ad, int *status, uint32_t *ber, uint16_t *strength,
-                   uint16_t *snr) {
+                   uint16_t *snr, uint16_t *db) {
 #if DVBAPIVERSION >= 0x050A
     *status = *snr = *ber = *strength = 0;
+    *db = 65535;
```

Comment:
use MAX_DB

Reply:
Hi @catalinii ,

> use MAX_DB

To do this I need to add `#include "adapter.h"` in "dvb.c". My idea is not to smudge the code. But if you prefer to do this change, then please agree with that.